go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/dave/jennifer v1.4.2-0.20211112003305-45cc0b7eb71a
 	github.com/julienschmidt/httprouter v1.3.0
 	github.com/nmiyake/pkg/dirs v1.1.0
-	github.com/palantir/conjure-go-runtime/v2 v2.83.0
+	github.com/palantir/conjure-go-runtime/v2 v2.83.1-0.20241107181000-fb10c9ca7ad8 // Requires https://github.com/palantir/conjure-go-runtime/pull/695
```

Comment:
We should cut a release for this before proceeding with this PR

Reply:
@nmiyake there's now a CGR release so this should be ready for review!